src/webhook_service/terraform/dynamodb.tf

```
@@ -8,6 +8,40 @@ resource "aws_dynamodb_table" "webhook" {
     type = "S"
   }
 
+  attribute {
+    name = "sequence_number"
+    type = "N"
+  }
+
+  attribute {
+    name = "webhook_type"
+    type = "S"
+  }
+
+  # The order of sequence_number is the same as the order of ascending order of created_at
+  # So sorting by sequence_number is the same as sorting by created_at, we only need to use SequenceNumberIndex to sort here.
+  global_secondary_index {
+    name               = "SequenceNumberIndex" # Updated GSI name
```

```suggestion
global_secondary_index {
    name = "webhook-webhook_type-sequence_number-gsi"

```

Apologies for not mentioning the naming guidelines for GSI earlier.  

Please use the following naming format:  
`{table_name}-{partition_key}-{sort_key}-gsi`  

Following this format will improve the code readability, making it easier for others to identify the table and understand the partition key (PK) and sort key (SK) of the GSI.  

For this case, you should rename the GSI to `webhook-webhook_type-sequence_number-gsi`.

---

The naming format has been applied to the global_secondary_index
```
  global_secondary_index {
    name               = "webhook-webhook_type-sequence_number-gsi" # Updated GSI name
    hash_key           = "webhook_type"        # Partition key
    range_key          = "sequence_number"     # Sort key
    projection_type    = "ALL"                 # Include all attributes in the GSI
  }
```
Take a look at the [dynamodb.tf](https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/terraform/dynamodb.tf)